### DISCUSSION TITLE
Viloxazine PBPK Model

### DISCUSSION THREAD

**Discussion by harishkaushikbugworks** [Open-Systems-Pharmacology/Forum#1795] [Questions & Problems]
Hello All,

Iam trying to build Viloxazine PBPK Model based on phys chem  and metabolism data available in literature, I have digitized IV and oral data from literature , somehow the model is not fitting well, it will help if one can look the attached model and help

Regards
Harish
[viloxazine pbpk model.ZIP](https://github.com/user-attachments/files/16141550/viloxazine.pbpk.model.ZIP)

**Comment by tobiasK2001** [comment]
Dear Harish,

I cannot find your model. It looks like the zip folder is empty.

**Reply by harishkaushikbugworks** [reply]
Sorry I missed checking the folder,
[PBPK Model.zip](https://github.com/user-attachments/files/16173340/PBPK.Model.zip)

attaching the model as zip file again

**Reply by tobiasK2001** [reply]
Thanks,

I found some basic technical errors in you model: 
1.) expression profiles of the enzymes were not set. I.e. relative expression in Organs were zero resulting in no clearance at all. It should look e.g. like this 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/a3b64ec8-de48-45a7-85cc-ad19f63aa7ca)
Data import was done wrong. Necessary Meta data was missing. It should look like this:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/838979fe-16aa-4e21-922d-b667fbe5ad63)
You also might to check the OSP documentation for how doing this correctly.
After that correction I followed the basic steps as outlined by Kuepfer et al (10.1002/psp4.12134). However it does not seem possible to get from a more or less OK IV profile
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/2647f0a9-7ccd-40ec-b598-54f31a641adf)
 to PO. 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/7302e07f-8446-4d03-b2f9-e8424eff8de7)
It looks like the model misses something for the absorption and some deeper investigation is needed. More in vivo data from other studies might be good.

Please find the model here. Good look!
[viloxazine_TK.zip](https://github.com/user-attachments/files/16174255/viloxazine_TK.zip)

**Reply by harishkaushikbugworks** [reply]
Thanks Tobias, for correcting the model

Regards
Harish
